packages/devextreme/testing/tests/DevExpress.viz.core/translator2D.tests.js

```diff
@@ -798,6 +798,19 @@ QUnit.test('Translate. Update translator with business range with empty scale br
     assert.strictEqual(translator.translate(450), 1000);
 });
 
+QUnit.test('Translate. Scale breaks. Translator should handle division by zero (T1222629)', function(assert) {
+    const breaks = [{ from: -11.9, to: -10.3, cumulativeWidth: 5 }, { from: -10.3, to: 20.3, cumulativeWidth: 10 }];
+    const translator = createTranslatorWithScaleBreaks.call(this, {});
+
+    translator.updateBusinessRange({ min: -11.9, max: 20.3, breaks: breaks });
+
+    const denominator = translator._canvasOptions.rangeMaxVisible - translator._canvasOptions.rangeMinVisible - translator._breaks[translator._breaks.length - 1].length;
```

Comment:
i believe we can just create translator (providing required parametrs, such as canvas and range ) , call updateBusinessRange and after that check translator.ratioOfCanvasRange(), why do we need such calculateions in a test? and also condition checking for zero seems redundant